ztp/tools/pgt2acmpg/scripts/build-acmpg-plugin.sh

```diff
@@ -4,16 +4,38 @@ set -o errexit -o nounset -o pipefail
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 CNF_FEATURE_DEPLOY_ROOT=$SCRIPT_DIR/../../../..
 
-# Builds the latest version of policy-generator-plugin
-# We can no longer use go install because of https://github.com/golang/go/issues/44840
-# Instead we need to clone the policy-generator-plugin project and build the executable
+# Due to https://github.com/golang/go/issues/44840 we are unable to use go install
+# Instead we will download the source code and compile it ourselves
 TMP_DIR=$(mktemp -d -p .)
-echo "Created $TMP_DIR"
-cd "$TMP_DIR"
-git clone --depth 1 --branch main --single-branch https://github.com/open-cluster-management-io/policy-generator-plugin.git
-cd policy-generator-plugin
+echo "Created ${TMP_DIR}"
+pushd "${TMP_DIR}"
+
+# The details we will use to query ACM
+ACM_FORK="stolostron"
```

Comment:
@edcdavid I did see that originally you had used open-cluster-management instead of stolostron, I am curious if you had a specific reason for that or not

Reply:
I guess it was my mistake, sorry about it